preprocess_courses.py

```diff
@@ -16,4 +16,4 @@
 
 print("Before")
 runner.preprocess()
-print("Done")
+print("Done")
```

Comment:
linter?

Reply:
Yeah not sure what happened there, I'll revert it!